### PR TITLE
Flowtest - mcg namespace disprution flow

### PR DESCRIPTION
**Pull request by parikshithb**
Signed-off-by: Parikshith <pbyregow@redhat.com>

### CODE REVIEW

**Review by ebenahar — DISMISSED** [review]

File: tests/e2e/flowtest/mcg/test_mcg_namespace_disruptions.py [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+        amount (Int): Number of test objects to create
+
+    """
+    awscli_pod.exec_cmd_on_pod(command=f'mkdir {MCG_NS_ORIGINAL_DIR} {MCG_NS_RESULT_DIR}')
+
+    for i in range(amount):
+        object_key = f"ObjKey-{i}"
+        awscli_pod.exec_cmd_on_pod(
+            f"dd if=/dev/urandom of={MCG_NS_ORIGINAL_DIR}/{object_key} bs=1M count=1 status=none"
+        )
```
Comment (PrasadDesala) [inline tests/e2e/flowtest/mcg/test_mcg_namespace_disruptions.py:45]: you are using the same code over here as well,
https://github.com/red-hat-storage/ocs-ci/blob/master/tests/e2e/lifecycle/mcg/test_mcg_namespace_lifecyle.py#L27-L42
can we create a common function?

File: tests/e2e/flowtest/mcg/test_mcg_namespace_disruptions.py [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+                    f"{awscli_node_name} is same as {pod_to_drain} "
+                    f"pod node: {node_name}"
+                )
+                continue
+
+            # Drain the node
+            drain_nodes([node_name])
+            wait_for_nodes_status([node_name], status=constants.NODE_READY_SCHEDULING_DISABLED)
+            schedule_nodes([node_name])
+
```
Comment (PrasadDesala) [inline tests/e2e/flowtest/mcg/test_mcg_namespace_disruptions.py:238]: post node drain, do you think we should also verify that all storage pods are in running state?

File: tests/e2e/flowtest/mcg/test_mcg_namespace_disruptions.py (hunk 1 above)
Comment (parikshithb) [reply tests/e2e/flowtest/mcg/test_mcg_namespace_disruptions.py]: ack

File: tests/e2e/flowtest/mcg/test_mcg_namespace_disruptions.py (hunk 2 above)
Comment (parikshithb) [reply tests/e2e/flowtest/mcg/test_mcg_namespace_disruptions.py:199]: ack

**Review by fbalak — APPROVED** [review]

**Review by shreekarSS — APPROVED** [review]
LGTM!!!

**Review by ebenahar — APPROVED** [review]

**Pull request merged**